pkg/kubectl/cmd/util/helpers.go

```diff
@@ -402,6 +402,7 @@ func AddDryRunFlag(cmd *cobra.Command) {
 func AddServerSideApplyFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool("server-side", false, "If true, apply runs in the server instead of the client. This is an alpha feature and flag.")
 	cmd.Flags().Bool("force-conflicts", false, "If true, server-side apply will force the changes against conflicts. This is an alpha feature and flag.")
+	cmd.Flags().String("field-manager", "kubectl", "Name of the manager used to track field ownership. This is an alpha feature and flag.")
```

Comment:
Shouldn't this be `experimental` because it's alpha?

Reply:
I'm not opposed to it, should I also change the two other flags? (also alpha)

Reply:
If they are still alpha, then yes.

Reply:
Yeah, I've updated them all